package.json

```diff
@@ -17,6 +17,8 @@
   "dependencies": {
     "@auth0/auth0-react": "1.3.0",
     "@aws-amplify/ui-react": "0.2.38",
+    "@cypress/react": "^5.1.0-alpha.1",
+    "@cypress/webpack-dev-server": "^1.0.4",
```

Comment:
@JessicaSachs @dmtrKovalenko Should these be moved to devDependencies?

Reply:
This doesn't really matter for apps right? Or you are using yarn install --production for server? 

But I don't care we can move it :)

Reply:
@dmtrKovalenko It does for vite tooling, which we will be merging soon as an option during development (https://github.com/cypress-io/cypress-realworld-app/pull/741).  Please move these to `devDependencies`.

Reply:
For Vite tooling, we'll be swapping this to the `@cypress/vite-dev-server` package. Trying out the webpack version first 😻 . We have an esm build as well, but yes, these should be devDeps.